packages/AdaptiveCards/AdaptiveCardComponentRegistration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Bot.Builder.Dialogs.Declarative.Obsolete;
+
+namespace Microsoft.Bot.Components.AdaptiveCards
+{
+    /// <summary>
+    /// <see cref="AdaptiveCardComponentRegistration"/> implementation for legacy runtimes.
+    /// </summary>
+    public class AdaptiveCardComponentRegistration 
```

Comment:
My understanding was that we should be able to get rid of this since this is only needed for existing dependencies on ComponentRegistration?

Reply:
This is to support legacy runtimes and PVA

---
In reply to: [610221302](https://github.com/microsoft/botframework-components/pull/862#discussion_r610221302) [](ancestors = 610221302)